### DISCUSSION TITLE
Can't make my app generate the PDF

### DISCUSSION THREAD

**Discussion by ThiagoMarvinSoares** [vercel/community#6393] [Help]
### Summary

My app generates a PDF from a html form, it works fine in localhost but when i set the vercel URL it gives me a 405 error

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/110858853/b72c4304-6dee-4681-a86c-e1186faa530c)

**Comment by amyegan** [comment]
Do you have a public [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that someone here could use to help you debug this error?

**Comment by ThiagoMarvinSoares** [comment]
I have it uploaded to vercel, https://geradordegarantia.vercel.app
And here is the base code https://github.com/ThiagoMarvinSoares/InvoicePDF
I think there is something with vercel and dowloading the file locally.
